### ISSUE TITLE
Insecure SSL Validation

### ISSUE THREAD

**Issue by Zyano** [braathen/qlik-auth-net#4]
Hello,

While this is a nice example and demonstrates who to do the authentication with Qlik I do believe that the following line:
`ServicePointManager.ServerCertificateValidationCallback = delegate { return true; };`
[Source](https://github.com/braathen/qlik-auth-net/blob/master/QlikAuthNet/QlikAuthNet.cs#L197 )

It is dangerous since it will affect all subsequent certificate validations, even for requests that are unrelated to the Qlik implementation.

**Comment by braathen** [comment]
I totally agree, it's a dumb way of doing it... If there's a desire to NOT validate the cert then it's probably a lot better to use HttpWebRequest.ServerCertificateValidationCallback on the specific request to not disturb others. Just need to find the time to look at this some day....
